server/db/migrations/20181113112523-create-application-comment.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('ApplicationComments', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      recruiterId: {
+        allowNull: false,
+        type: Sequelize.INTEGER
+      },
+      jobApplicationId: {
+        allowNull: false,
+        type: Sequelize.INTEGER
+      },
+      attachmentId: {
```

Comment:
Tämä on minun vika, kun ajatukset ei eilen ollut ihan kasassa. Mutta tulevaan liite-tauluunhan ei tarvitakaan kommentista linkkiä, vaan nimenomaan toisinpäin: kommenttiinhan voi liittyä useampia liitteitä